### ISSUE TITLE
Disable button example of GDS fails colour contrast

### ISSUE THREAD

**Issue by Terry-Price** [alphagov/govuk-design-system-backlog#302]
## Disabled button colour contrast failure
> As the transparence of 50% is applied to the background colour on the GDS disabled button, it effectively is this colour #8EBAA4,

This colour with white text delivers a colour contrast ration of 2.16.1. 
It fails at every level and size.

![contrast checker failure](https://github.com/user-attachments/assets/1702d089-d33f-4012-96cc-b88ea3021fce)

**Comment by querkmachine** [comment]
Hi @Terry-Price,
This is not a WCAG failure, as per the [understanding minimum contrast guide](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html#inactive-controls):

> User Interface Components that are not available for user interaction (e.g., a disabled control in HTML) are not required to meet contrast requirements. An inactive user interface component is visible but not currently operable. An example would be a submit button at the bottom of a form that is visible but cannot be activated until all the required fields in the form are completed.

The same applies to the contrast with the button's background and the page background in the [understanding non-text contrast guide](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast#inactive-controls), which further elaborates on why they don't require contrast for inactive controls.

While it can be argued that it'd be more accessible for disabled controls to still meet minimum contrast requirements, doing so runs the risk of a disabled control looking too similar to an enabled one. We [advise against using disabled controls in user interfaces the first place](https://design-system.service.gov.uk/components/button/#disabled-buttons) to avoid the potential for confusion.
